tumblr/XExtensionItem
# Adds support for swift package manager

## What it does

- Adds a `Package.swift` file with 2 targets.
- Moves the folder structure to be compatible with SPM.
- Updates `.podspec` and `Podfile`
- Ran pod install in the example project.

## How to test

- Open the `Package.swift` file and make sure it builds successfully.
- Build and run the example project using the Photos app.
- Share a photo to `XExtensionItemExample`.

<img src="https://github.com/tumblr/XExtensionItem/assets/5721324/f46b3783-0824-479f-b8fa-c6314d842b24" width="250" height="445" />
